plasmapy/atomic/ionization_state.py

```diff
@@ -455,43 +497,64 @@ def equilibrate(self, T_e: u.K = None):
         # self.ionic_fractions = self.equil_ionic_fractions
         raise NotImplementedError
 
-    @property
-    def atomic_number(self) -> int:
-        """Return the atomic number of the element."""
-        return self._particle.atomic_number
+    def _is_normalized(self, tol: Optional[Real] = None) -> bool:
+        """
+        Return `True` if the sum of the ionization fractions is equal to
+        one within the allowed tolerance, and `False` otherwise.
+        """
+        tol = tol if tol is not None else self.tol
+        if not isinstance(tol, Real):
+            raise TypeError("tol must be an int or float.")
+        if not 0 <= tol < 1:
+            raise ValueError("Need 0 <= tol < 1.")
+        total = np.sum(self._ionic_fractions)
+        return np.isclose(total, 1, atol=tol, rtol=0)
+
+    def normalize(self) -> None:
+        """
+        Normalize the ionization state distribution (if set) so that the
+        sum becomes equal to one.
+        """
+        self._ionic_fractions = self._ionic_fractions / np.sum(self._ionic_fractions)
 
     @property
-    def element(self) -> str:
+    def _element(self) -> str:
         """Return the atomic symbol of the element."""
-        return self._particle.element
+        return self._particle_instance.element
```

Comment:
I'm not quite sure why you would make this a private-ish property.

Reply:
I created the `base_particle` attribute to return the element or isotope that an `IonizationState` instance is keeping track of, and to be more general.  On the other hand, there may be times when the element in particular is needed to be known, like to look up stuff in an atomic database which is broken down by element.  I'll go ahead and de-privatize this.